api/types/plugin.go

```diff
@@ -42,6 +42,10 @@ type PluginConfig struct {
 	// Required: true
 	Description string `json:"Description"`
 
+	// Docker Version used to create the plugin
+	// Required: true
+	DockerVersion string `json:"DockerVersion"`
```

Comment:
I dont feel too strongly either way. This case is applicable only for plugins built using 17.03/17.04 and running on versions >= 17.05.